lib/pluginmanager/templates/output-plugin/README.md

```diff
@@ -0,0 +1,86 @@
+# Logstash Plugin
```

Comment:
agreed, also it would be nice to have more specific readmes, our readme looks too general end of the day. but as you said, this is a matter of another PR.